indico/modules/events/registration/models/items.py

```diff
@@ -97,6 +132,22 @@ class RegistrationFormSection(RegistrationFormItem):
         'polymorphic_identity': RegistrationFormItemType.section
     }
 
+    @property
+    def locator(self):
+        return dict(self.registration_form.locator, section_id=self.id)
+
+    @property
+    def angular_data(self):
+        return {
+            'id': self.id,
+            'title': self.title,
+            'description': self.description,
+            'items': [child.angular_data for child in self.children if not child.is_deleted],
+            'enabled': self.enabled,
+            '_type': 'GeneralSectionForm',
```

Comment:
At the moment Angular uses `_type` to render different sections. However, it is going to be removed due to the fact that we will have only GeneralSection without any specific types. But first, I need to adapt Angular so that it doesn't use this field.